RepoDb.Core/RepoDb/ClassProperty.cs

```diff
@@ -307,9 +307,9 @@ public string GetMappedName()
         /// <returns>True if the two instance is the same.</returns>
         public override bool Equals(object obj)
         {
-            if (obj is ClassProperty)
+            if (obj is ClassProperty property)
```

Comment:
I am using this approach at some point, but when I integrated with App Veyor using the earlier version of builder, it fails. But do not worry, I already had updated the CI and it works with this approach. Thanks for this update.